docs/content/guides/getting-started/introduction.md

```diff
@@ -27,24 +27,24 @@ Explore Handsontable's features and source code:
 ## Guides for your framework
 
 <div class="row-items-container">
-  <Link href="../javascript-data-grid/" hide-latest-version class="row-item">
-    <Img class="integration-framework-logo" src="$withBase('/img/pages/introduction/javascript.svg')" alt="JavaScript logo" />
+  <Link href="/docs/{docsVersion}/binding-to-data/" hide-latest-version class="row-item">
+    <Img class="integration-framework-logo" src="/img/pages/introduction/javascript.svg" alt="JavaScript logo" />
```

Comment:
The `<Img>`'s `src` attribute is always treated as relative to `base`.  I'm not sure if that's correct, but I'm having trouble finding examples of this being problematic.

That's different from the native markdown's image and link syntax, where `{{$basePath}}` needs to be provided, though. Shouldn't they work more similarly?

Reply:
When you change the `src` attribute to an absolute path like `https://handsontable.com/docs/img/pages/introduction/vue.svg`, the `Img` component will still be rendered correctly. Nothing needs to be done in this case, I think.